internal/policies/apparmor/apparmor.go

```diff
@@ -229,17 +233,88 @@ func (m *Manager) ApplyPolicy(ctx context.Context, objectName string, isComputer
 	if err := os.RemoveAll(oldApparmorPath); err != nil {
 		return fmt.Errorf(i18n.G("can't remove old apparmor directory %q: %v"), oldApparmorPath, err)
 	}
+	return nil
+}
+
+// applyUserPolicy applies apparmor policies for the user object.
+func (m *Manager) applyUserPolicy(ctx context.Context, e entry.Entry, apparmorPath string, username string, assetsDumper AssetsDumper) (err error) {
+	defer decorate.OnError(&err, i18n.G("can't apply user policy"))
+
+	// Create a temporary directory to store the assets
```

Comment:
// Use
:) I was going to say "but you are defering too late! when I realize you just get a name here…)